index.html

```diff
@@ -41,7 +41,7 @@ <h3>how are you feeling now?</h3>
         <section id="layer-select"></section>
       </main>
       <section id="tool-options"></section>
-      <section id="tool-info"></section>
+      <section id="tool-info"> <h4> hint: scroll further than u think</h4></section>
```

Comment:
are we sure we want this here? remember that changes to the `index.html` page will effect all the themes. also the `tool-info` section is being dynamically modified by the framework